### PR TITLE
Spanish translation of the file preprocessing.mdx

### PR DESCRIPTION
**Pull request by yharyarias**
Hi, I made the translation of the **preprocessing.mdx** file to add it to the Spanish documentation (transformers/docs/source_es/). 
I will be waiting for any correction, thanks.
@omarespejel @LysandreJik @sgugger

### CODE REVIEW

**Comment by sgugger** [comment]
All good to me from the syntax point of view. Be careful though, as you PR files include the `training.mdx` so we can't know if there are small changes in it.

**Comment by yharyarias** [comment]
You're right, it was a mistake, It's just the preprocessing file

**Comment by omarespejel** [comment]
Hi @yharyarias! Could you please review what Sylvain[ commented above](https://github.com/huggingface/transformers/pull/16299#issuecomment-1074021882)? We are almost done.

File: docs/source_es/preprocessing.mdx [hunk 1]
```diff
@@ -0,0 +1,558 @@
...
+
+* `array` es la señal de voz cargada - y potencialmente resampled - como un array 1D.
+* `path` apunta a la ubicación del archivo de audio.
+* `sampling_rate` se refiere a cuántos puntos de datos de la señal de voz se miden por segundo.
+
+### Resample
+
+Para este tutorial, se utilizará el modelo [Wav2Vec2](https://huggingface.co/facebook/wav2vec2-base). Como puedes ver en la model card, el modelo Wav2Vec2 está preentrenado en audio de habla aplicando sampled a 16kHz. Es importante que el sampling rate de tus datos de audio coincida con el sampling rate del dataset utilizado para preentrenar el modelo. Si el sampling rate de tus datos no es el mismo, deberás volver aplicar resample tus datos de audio. 
+
+Por ejemplo, carga el dataset [LJ Speech](https://huggingface.co/datasets/lj_speech) que tiene un sampling rate de 22050kHz. Para utilizar el modelo Wav2Vec2 con este dataset, reduce el sampling rate a 16kHz:
```
Comment (omarespejel) [inline docs/source_es/preprocessing.mdx:222]:
```suggestion
Por ejemplo, carga el dataset [LJ Speech](https://huggingface.co/datasets/lj_speech) que tiene una tasa de muestreo de 22050kHz. Para utilizar el modelo Wav2Vec2 con este dataset, reduce la tasa de muestreo a 16kHz:
```

File: docs/source_es/preprocessing.mdx [hunk 2]
```diff
@@ -0,0 +1,558 @@
...
+```py
+>>> lj_speech = load_dataset("lj_speech", split="train")
+>>> lj_speech[0]["audio"]
+{'array': array([-7.3242188e-04, -7.6293945e-04, -6.4086914e-04, ...,
+         7.3242188e-04,  2.1362305e-04,  6.1035156e-05], dtype=float32),
+ 'path': '/root/.cache/huggingface/datasets/downloads/extracted/917ece08c95cf0c4115e45294e3cd0dee724a1165b7fc11798369308a465bd26/LJSpeech-1.1/wavs/LJ001-0001.wav',
+ 'sampling_rate': 22050}
+```
+
+1. Usa el método 🤗 Datasets' [`cast_column`](https://huggingface.co/docs/datasets/package_reference/main_classes.html#datasets.Dataset.cast_column) para reducir el sampling rate a 16kHz:
```
Comment (omarespejel) [inline docs/source_es/preprocessing.mdx:233]:
```suggestion
1. Usa el método 🤗 Datasets' [`cast_column`](https://huggingface.co/docs/datasets/package_reference/main_classes.html#datasets.Dataset.cast_column) para reducir la tasa de muestreo a 16kHz:
```

File: docs/source_es/preprocessing.mdx [hunk 3]
```diff
@@ -0,0 +1,558 @@
...
+>>> dataset["train"][0]["audio"]
+{'array': array([ 0.        ,  0.        ,  0.        , ..., -0.00592041,
+        -0.00405884, -0.00253296], dtype=float32),
+ 'path': '/root/.cache/huggingface/datasets/downloads/extracted/05734a36d88019a09725c20cc024e1c4e7982e37d7d55c0c1ca1742ea1cdd47f/_background_noise_/doing_the_dishes.wav',
+ 'sampling_rate': 16000}
+```
+
+Esto devuelve tres elementos:
+
+* `array` es la señal de voz cargada - y potencialmente resampled - como un array 1D.
```
Comment (omarespejel) [inline docs/source_es/preprocessing.mdx:214]:
```suggestion
* `array` es la señal de voz cargada - y potencialmente remuestreada - como un array 1D.
```

File: docs/source_es/preprocessing.mdx [hunk 4]
```diff
@@ -0,0 +1,558 @@
...
+
+<Tip>
+
+Si tienes previsto utilizar un modelo preentrenado, es importante que utilices el tokenizador preentrenado asociado. Esto te asegura que el texto se divide de la misma manera que el corpus de preentrenamiento, y utiliza el mismo índice de tokens correspondiente (usualmente referido como el *vocab*) durante el preentrenamiento.
+
+</Tip>
+
+Comienza rápidamente cargando un tokenizador preentrenado con la clase [`AutoTokenizer`]. Esto descarga el *vocab* utilizado cuando un modelo es preentrenado.
+
+### Tokenize
```
Comment (omarespejel) [inline docs/source_es/preprocessing.mdx:37]:
```suggestion
### Tokenizar
```

**Review by omarespejel — CHANGES_REQUESTED** [review]
Thank you for your PR! It seems that your translation includes text that is not in the original English version (lines 491 onwards). I will eliminate them for now. Will also eliminate the training.mdx translation in this PR.

**Review by omarespejel — APPROVED** [review]

**Comment by omarespejel** [comment]
Muchas gracias @yharyarias for the PR! 🤗 Please let me know if you wish to translate another one. 

@sgugger LGTM :)

**Review by sgugger — APPROVED** [review]
Thanks for adding this new translation!

**Pull request merged**